policyengine_us/parameters/gov/states/ri/tax/income/credits/property_tax/max_amount.yaml

```
@@ -0,0 +1,22 @@
+description: Rhode Island provides the following maximum amount under the property tax credit.
+
+values:
+  1977-01-01: 55
+  1978-01-01: 150
+  1979-01-01: 175
+  1980-01-01: 200
+  1997-01-01: 250
+  2006-01-01: 300
+  2007-01-01: 500
+  2022-01-01: 600
+metadata:
+  label: Rhode Island property tax credit maximum amount
+  period: year
+  unit: currency-USD
+  reference:
+  - title: Property Tax Relief R.I. Gen. Laws § 44-33-9 (2)
+    href: http://webserver.rilin.state.ri.us/Statutes/TITLE44/44-33/44-33-9.htm
+  - title: 2022 Form RI-1040H Rhode Island Property Tax Relief Claim
```

Add line numbers elsewhere
```suggestion
  - title: 2022 Form RI-1040H Rhode Island Property Tax Relief Claim Line 12
```